credmark/model/describe.py

```diff
@@ -161,18 +161,14 @@ def is_parent(child, parent):
         else:
             validate_model_slug(slug)
 
-        model_desc = description
```

Comment:
Why did you remove this?

Reply:
The merging was not done properly. I have restored.